felix/bpf-gpl/tc6.c

```diff
@@ -24,7 +24,7 @@
 const volatile struct cali_tc_globals __globals;
 
 
-SEC("classifier/tc/prologue")
+SEC("classifier")
```

Comment:
Why did we need to place everything into a single section now?

Reply:
The way the section name is parsed has changed. In earlier versions, ```classifier/x/y``` was allowed but in the latest it is just ```classifier```.

Reply:
I have also changed the section name from ```classifier``` to ```tc``` as ```classifier``` is deprecated from v1.0.0